lib/views/teach_word_view.dart

```diff
@@ -0,0 +1,356 @@
+import 'package:flutter/material.dart';
+import 'package:ltrc/extensions.dart';
+import 'package:flutter/services.dart' show rootBundle;
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:provider/provider.dart';
+import 'package:ltrc/widgets/teach_word/tab_bar_view.dart';
+import 'package:ltrc/main.dart';
+import 'package:flutter_tts/flutter_tts.dart';
+import 'package:ltrc/widgets/mainPage/left_right_switch.dart';
+import 'package:ltrc/widgets/teach_word/card_title.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animation_controller.dart';
+import 'package:ltrc/widgets/teach_word/stroke_order_animator.dart';
+
+class TeachWordView extends StatefulWidget {
+  const TeachWordView({super.key});
+
+  @override
+  State<TeachWordView> createState() => _TeachWordViewState();
+}
+
+class _TeachWordViewState extends State<TeachWordView>
+    with TickerProviderStateMixin {
+  late StrokeOrderAnimationController _strokeOrderAnimationControllers;
+  late TabController _tabController;
+  FlutterTts ftts = FlutterTts();
+
+  String svg = "";
+
+  Future<String> readJson() async {
+    final String response =
+        await rootBundle.loadString('lib/assets/svg/' + demoChar + '.json');
+
+    await ftts.setLanguage("zh-tw");
+    return response.replaceAll("\"", "\'");
+  }
+
+  @override
+  void initState() {
+    _tabController = TabController(length: 4, vsync: this);
+    super.initState();
+    readJson().then((result) {
+      setState(() {
+        _strokeOrderAnimationControllers = StrokeOrderAnimationController(
+          result,
+          this,
+          onQuizCompleteCallback: (summary) {
+            Fluttertoast.showToast(
+                msg: [
```

Comment:
Align `msg:[ ....` with its counterpart `]`

Reply:
"Format On Save” did that, I don't know how to align it manually.